.github/workflows/update-man-pages.yml

```
@@ -0,0 +1,39 @@
+name: Update man-pages workflow
+on:
+  schedule:
+    - cron: "42 3 1/15 * *"   # Trigger every 15 days at 03:42
+    #- cron: "0,5,10,15,20,25,30,35,40,45,50,55 * * * *"  # For testing
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-man-pages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository code
+        uses: actions/checkout@v4
+
+      - name: Update man-pages
+        run: |
+          set -euo pipefail
+
+          rm ./documentation/manpages/sdk/*
+
+          ./documentation/manpages/tool/run_docker.sh
+
+          if [[ -n $(git status -s) ]]; then
+            git config user.name 'TODO'
+            git config user.email 'TODO@example.com'
+            date=$(date +%F)
+            git add .
+            title="Update man pages"
+            branch=update-man-page-$date
+            git checkout -b $branch
+            git commit -a -m "$title"
+            git push -u origin $branch --force
+            gh pr create --base main --head $branch --title "$title" --body "Auto-generated"
```

Maybe we could add an `[automated]` prefix to the title for extra clarity?

Maybe we could add some additional details to the PR body?

```suggestion
            gh pr create --base main --head $branch --title "[automated] $title" --body "This action is executed twice a month to automatically update the manpages based on latest changes to the repo"
```

---

Thanks. Done.